File: GigsApplication/Core/Models/Comment.cs
namespace GigsApplication.Core.Models
{
    public class Comment
    {
        public int Id { get; set; }
        public string comment { get; set; }
        public ApplicationUser User { get; set; }
        public string UserId { get; set; }
        public Gig Audio { get; set; }
        public int AudioId { get; set; }
    }
}

File: GigsApplication/Scripts/app/user/comment.js
var Comment = function (commentService) {
    var message;
    var name;
    var click = function () {
        $(".js-toggle-comment").click(comment);
    }
    var comment =  function (e) {
        button = $(e.target);
        var audioId = button.attr("data-audio-id");
        name = button.attr("data-user-name");
        message = $(".comment-data").val();

        commentService.Postcomment(audioId, message,done,fail);
    }
    var done = function () {
        $(".comment-data").val('');
        var comment = `<div class="comment"><span id="commented-user">${name}</span><p>${message}</p></div>`
        $(".comments-container").append(comment);
    }
    var fail = function () {
        alert("some thing gone Wrong!");
    }
    return {
    clickComment:click
    }

}(CommentService);